documentation/book/cluster-operator.adoc

```diff
@@ -546,25 +561,23 @@ More information about these configuration parameters in the related <<Topic Ope
 In order to configure a Kafka Connect cluster deployment, it's possible to specify the following fields in the `data` section of 
 the related ConfigMap:
 
-`nodes`:: Number of Kafka Connect worker nodes. Default is 1
-`image`:: The Docker image to use for the Kafka Connect workers.
-Default is determined by the value of the
-`<<STRIMZI_DEFAULT_KAFKA_CONNECT_IMAGE,STRIMZI_DEFAULT_KAFKA_CONNECT_IMAGE>>`
-environment variable of the Cluster Operator.
-If S2I is used (only on OpenShift), then it should be the related S2I image.
-`healthcheck-delay`:: The initial delay for the liveness and readiness probes for each Kafka Connect worker node. Default is 60
-`healthcheck-timeout`:: The timeout on the liveness and readiness probes for each Kafka Connect worker node. Default is 5
+* `nodes`: Number of Kafka Connect worker nodes. Default is 1.
```

Comment:
Why here are we going from `` `foo`:: blah `` → ``* `foo`: blah ``, when the other lists in the document are (mostly) using the former syntax?